### PR TITLE
📝 Use in memory database for testing SQL in docs

### PR DESCRIPTION
**Pull request by HarshaLaxman**
Useful for me, maybe it is better as just a note?

### CODE REVIEW

File: docs/en/docs/advanced/testing-database.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ So the new file structure looks like:
 
 First, we create a new database session with the new database.
 
-For the tests we'll use a file `test.db` instead of `sql_app.db`.
+We'll use an in memory database that persists for the duration of the tests.
```
Comment (Kludex) [inline docs/en/docs/advanced/testing-database.md:47]: Maybe?
```suggestion
We'll use an in memory database that persists during the tests.
```
LGTM overall :+1:

**Review by Kludex — APPROVED** [review]

**Comment by eboddington** [comment]
I think this would be helpful to merge, or at least to have as a side note. I've wasted a while trying to get an in-memory database working for tests, only to find this and realise that I was missing `poolclass=StaticPool`!

**Review by jnnnnn — APPROVED** [review]

**Review by yezz123 — APPROVED** [review]

**Review by BilalAlpaslan — APPROVED** [review]

**Review by iudeen — APPROVED** [review]

**Comment by davidhammaker** [comment]
I don't know whether this will be merged (seems to be open still?), but it has helped me. It saves a lot of clean up during and after tests.

**Comment by HarshaLaxman** [comment]
@tiangolo would you like to merge?

**Comment by tiangolo** [comment]
Thanks @HarshaLaxman! 🚀 

I've been wanting to update the docs about SQL to use SQLModel and re-use a bunch of those docs there. But for now, I'll take this. 🤓 

And thanks everyone else for the comments and for letting me know that this is useful to you! 🍰

**Pull request merged**